static/src/javascripts/bootstraps/atoms/storyquestions.js

```diff
@@ -23,13 +35,15 @@ Promise.all([
     new Promise(comready),
 ]).then(() => {
     init();
-    fastdom
-        .read(
-            () =>
-                document.documentElement &&
-                document.documentElement.getBoundingClientRect().height
-        )
-        .then(height => {
-            send('resize', { height });
+    updateHeight();
+
+    // Brittle but will work
+    [...document.getElementsByClassName('user__question')]
```

Comment:
I'd probably use `Array.from` instead, that utility function can be improved ... if only it had range parameters.